programmers/이상한-문자-만들기.js

```diff
@@ -1,12 +1,12 @@
 function upperToLowerLowerToUpper(s) {
```

Comment:
함수는 한번에 하나의 기능만 하도록 구현해야하는 것 아실거예요.
함수 이름에서 두가지의 기능을 하는 늬앙스를 많이 풍기고 있어서 아래처럼 바꿔주는게 더 좋을 것 같아요.
```suggestion
function convertUpperAndLower(s) {
```